core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -765,6 +766,43 @@ protected function doReplace($strBuffer, $blnCache)
 					break;
 
 				// Images
+				case 'figure':
+					// Expected format: {{figure::<from>[?<key>=<value>,[&<key>=<value>]*]}}
+					list($from, $configuration) = $this->parseUrlWithQueryString($elements[1] ?? '');
+
+					if (null === $from || 2 !== \count($elements))
+					{
+						$arrCache[$strTag] = '';
+
+						break;
+					}
+
+					// Build arguments for FigureRendererRuntime#render
+					$size = $configuration['size'] ?? null;
+					$template = $configuration['template'] ?? null;
```

Comment:
This only supports twig templates not Contao html5 templates, right?

Reply:
It's now refactored and all three usages accept both.